### DISCUSSION TITLE
I wonder if the content within .classes() refers to Quasar class names or Tailwind class names? I can't find a precise explanation anywhere.

### DISCUSSION THREAD

**Discussion by zouzhe1** [zauberzeug/nicegui#3391] [Q&A]
### Question

I wonder if the content within **.classes()** refers to **Quasar** class names or **Tailwind** class names? I can't find a precise explanation anywhere.

Because I found that sometimes it's not easy to use the Quasar class name, and sometimes it's not easy to use the Tailwind class name, so I got confused.

**Comment by python-and-novella** [comment]
Use search in https://quasar.dev/ and https://tailwindcss.com/
Or see it in： 
How to simplify styling? · zauberzeug/nicegui · Discussion #117
https://github.com/zauberzeug/nicegui/discussions/117
Have a try by `Ctrl+K` or spend more time in learning carefully.
![image](https://github.com/user-attachments/assets/4acd2ed8-1548-400e-b572-6ec8e2d354cb)
![image](https://github.com/user-attachments/assets/04e2075f-596d-453f-a7d9-d91b2f9359ff)

**Comment by falkoschindler** [comment]
The `.classes()` method adds `class` attributes to the generated Vue or HTML element. It can be Tailwind, Quasar or other class names - it's up to you. And yes, sometimes Tailwind and Quasar classes interfere with each other. That's an unfortunate consequence of using two slightly overlapping frontend frameworks.
